src/miner.cpp

```diff
@@ -473,6 +473,9 @@ void static BitcoinMiner(CWallet *pwallet)
     );
 
     try {
+        unique_ptr<equi> peq;
+        if (solver == "tromp")
+            peq.reset(new equi(1));
```

Comment:
you mean we can put it back inside the cycle?

anyway in this case noone free the memory allocated for equi (not a big deal but still...).